models/organization.go

```diff
@@ -186,6 +186,28 @@ func (om *OrgManager) GetTotalStorageUsed(name string) (uint64, error) {
 	return total, nil
 }
 
+// GetUserUploads is used to return all uploads from the organization user
+func (om *OrgManager) GetUserUploads(orgName, username string) ([]Upload, error) {
+	_, err := om.FindByName(orgName)
+	if err != nil {
+		return nil, err
+	}
+	usr, err := NewUserManager(om.DB).FindByUserName(username)
+	if err != nil {
+		return nil, err
+	}
+	// prevent returning upload results for users that are not part of the organization
+	// this is to prevent exploiting users from examining other peoples data, however
+	// we still need to implement an auth check in the caller.
+	if usr.Organization != orgName {
+		return nil, errors.New("user does not belong to organization")
+	}
```

Comment:
since we are checking for orgName matching, I don't think the first `om.FindByName(orgName)` is needed

Reply:
If we do that, and an empty organization name is given, then it's possible for the caller to retrieve uploads of all non organization users

Reply:
Granted we will prevent this with a check in the caller within Temporal's API, I thinkt this is a good check to have an avoid accidentally disclosing data.

Reply:
ok